test/unified_format.py

```diff
@@ -1087,6 +1087,25 @@ def _databaseOperation_runCommand(self, target, **kwargs):
         kwargs["command"] = ordered_command
         return target.command(**kwargs)
 
+    def _databaseOperation_runCursorCommand(self, target, **kwargs):
+        self.__raise_if_unsupported("runCursorCommand", target, Database)
+        # Ensure the first key is the command name.
+        ordered_command = SON([(kwargs.pop("command_name"), 1)])
+        ordered_command.update(kwargs["command"])
+        kwargs["command"] = ordered_command
+        if "maxTimeMS" in kwargs:
+            kwargs["max_time_ms"] = kwargs["maxTimeMS"]
+            del kwargs["maxTimeMS"]
+        return list(target.cursor_command(**kwargs))
+
+    def _databaseOperation_createCommandCursor(self, target, **kwargs):
+        self.__raise_if_unsupported("createCommandCursor", target, Database)
+        # Ensure the first key is the command name.
+        ordered_command = SON([(kwargs.pop("command_name"), 1)])
+        ordered_command.update(kwargs["command"])
+        kwargs["command"] = ordered_command
+        return target.cursor_command(**kwargs)
```

Comment:
Let's add support the cursorType field for now (while DRIVERS-2664 is discussed). Like:
```
ctype = kwargs.pop("cursor_type", "nonTailable")
if ctype == "tailable":
    ordered_command["tailable"] = True
elif ctype == "tailableAwait":
    ordered_command["tailable"] = True
    ordered_command["awaitData"] = True
elif ctype != "nonTailable"
    self.fail(f"unknown cursorType: {ctype}")
```